### PR TITLE
People details

### PR DESCRIPTION
**Pull request by m4dzix**

### CODE REVIEW

File: src/common/Main/style.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import styled from "styled-components";
 
 export const StyledMain = styled.main`
-  display: flex;
+  display: grid;
```
Comment (KacperSzemraj) [inline src/common/Main/style.js:4]: Ok, good move!

File: src/common/Tile/DetailsTile/styled.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  grid-template-columns: auto 1fr;
+  background-color: ${({ theme }) => theme.colors.white};
+  align-items: start;
+`;
+
+export const Poster = styled.img`
+  width: 312px;
+  height: 464px;
+  border-radius: 5px;
+  background-color: #eee;
```
Comment (KacperSzemraj) [inline src/common/Tile/DetailsTile/styled.js]: You can add this color to the theme ;)

File: src/common/Tile/DetailsTile/styled.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  max-width: 1368px;
+  padding: 16px;
+  box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
```
Comment (KacperSzemraj) [inline src/common/Tile/DetailsTile/styled.js]: If you think there should be a shadow, let's leave it that way, but there is no such thing in Figma. ;)

File: src/common/Tile/DetailsTile/styled.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  font-size: 18px;
+  line-height: 120%;
+  margin: 0;
+  padding: 0;
+  margin-bottom: 8px;
+`;
+export const Span = styled.span`
+  font-size: 18px;
+  line-height: 120%;
+  color: #74788b;
```
Comment (KacperSzemraj) [inline src/common/Tile/DetailsTile/styled.js]: We can add this color to the theme ;)

**Review by KacperSzemraj — CHANGES_REQUESTED** [review]
Hello m4dzix!

The tile with the details of the person has no distance from the navigation bar. Of course this space will also be visible on other subpages of the movies-browser.

We will have to think about adding to the main section padding-top: 56px. For mobile: 24px;
By the way, it is also worth noting the removal of the default links style on the People page.

Something strange is also happening in the mobile view; namely the distance between the photo of a person and the edge of a tile. ;) There is some margin, margin-top: -6px;

Have a nice day! Kacper

File: src/common/Tile/DetailsTile/styled.js (hunk 2 above)
Comment (m4dzix) [reply src/common/Tile/DetailsTile/styled.js]: Wow, right!. What a stupid mistake 😆

File: src/common/Tile/DetailsTile/styled.js (hunk 3 above)
Comment (m4dzix) [reply src/common/Tile/DetailsTile/styled.js]: You're right - it's not in Figma - weird. Some tiles have a shadow and some do not. Maybe I'll leave it, it looks good to me 😜

**Review by KacperSzemraj — APPROVED** [review]
Hi!

Great, thanks for the changes! ;)

Kacper

**Pull request closed without merging**